packages/blockchain-wallet-v4-frontend/src/scenes/Exchange/GetStarted/StatusBar/selectors.js

```diff
@@ -0,0 +1,19 @@
+import { selectors } from 'data'
+import { KYC_STATES, USER_ACTIVATION_STATES } from 'data/modules/profile/model'
+
+export const getData = state => {
+  const userState = selectors.modules.profile.getUserActivationState(state).getOrElse(null)
+  const kycState = selectors.modules.profile.getUserKYCState(state).getOrElse(null)
+
+  if (userState === USER_ACTIVATION_STATES.NONE) {
+    return { step: 'getstarted' }
+  } else {
```

Comment:
else is not needed in case of if ... return
will reduce complexity by 1 level

Reply:
Yep